src/GitHub/Endpoints/Repos/Traffic.hs

```diff
@@ -0,0 +1,82 @@
+{-# LANGUAGE LambdaCase #-}
```

Comment:
I probably wasn't clear enough, could you not use `LambdaCase`.

Seems that there's only one other module using it, which I'll clean up.

I simply don't like LambdaCase

Reply:
> I simply don't like LambdaCase

I do, very much so!